resources/js/bootstrap.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
this file is not needed if Inertia will be used.

Reply:
removed